database/migrations/20170531173912_organizations.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = knex =>
+  knex.schema.createTable( 'organizations', table => {
+    table.increments( 'id' ).primary()
+    table.integer( 'user_id' ).notNullable()
```

Comment:
ok that makes sense to change it to owner...so we should probably change `user_name` to `owner_name` too right?